tests/test_clients.py

```diff
@@ -99,10 +99,10 @@
             "formatted_citation": (
                 "Andres M. Bran, Sam Cox, Oliver Schilter, Carlo Baldassari, Andrew D."
                 " White, and Philippe Schwaller. Augmenting large language models with"
-                " chemistry tools. Nature Machine Intelligence, 6:525-535, May 2024."
+                " chemistry tools. Nature Machine Intelligence, 6:535, May 2024."
```

Comment:
This citation is wrong - the starting page should be 525, not 535. I guess this is cassetted?